components/Listen/EpisodeList/EpisodeList.tsx

```diff
@@ -54,6 +54,7 @@ const EpisodeList: React.FC<IEpisodeListProps> = ({
             index={index}
             onEpisodeClick={onEpisodeClick}
             key={track.guid}
+            title="test"
```

Comment:
Looks like some debugging code was left behind. This needs to be removed. It is causing build errors.